src/SprykerEco/Shared/Unzer/Transfer/unzer.transfer.xml

```diff
@@ -267,9 +267,9 @@
     </transfer>
 
     <transfer name="UnzerCredentialsCriteria">
-        <property name="unzerCredentialsConditions" type="UnzerCredentialsConditions" />
+        <property name="unzerCredentialsConditions" type="UnzerCredentialsConditions"/>
```

Comment:
As discussed, we should really have singular in the singular property so that we get something like addUnzerCredentialsCondition and so on.